lessons/misc/jekyll-ghpages/lesson.md

```diff
@@ -23,7 +23,7 @@ In this code-along we'll cover what GitHub Pages are, why you might want to use
 
 - GitHub account (if you don't already have one go to [github.com/join](https://github.com/join))
 - git installed, and configured on your machine ([git-scm.com](https://git-scm.com/downloads))
-- comfort with material covered in UofT Coders lessons **[Intro to Git](https://uoftcoders.github.io/studyGroup/lessons/git/lesson/)**, **[More on Git](https://uoftcoders.github.io/studyGroup/lessons/git2/lesson/)**, and **[Using Branches in Git](https://uoftcoders.github.io/studyGroup/lessons/gitBranches/lesson/)**
+- comfort with material covered in UofT Coders lessons **[Intro to Git](https://uoftcoders.github.io/studyGroup/lessons/git/intro/lesson/)**, **[More on Git](https://uoftcoders.github.io/studyGroup/lessons/git/advanced/lesson/)**, and **[Using Branches in Git](https://uoftcoders.github.io/studyGroup/lessons/git/branches/lesson/)**
```

Comment:
Only thing here is the `More on Git` should be `Intermediate Git` or something. To match your new change in the actual file.